Formula/rack.rb

```diff
@@ -23,9 +23,16 @@ def install
     rackpath.install Dir["{*,.??*}"]
 
     cd rackpath do
+      # This is a slightly grim hack to handle the weird logic around
+      # deciding whether to add a = or not on the ldflags, as mandated
+      # by Go 1.7+.
+      # https://github.com/rackspace/rack/issues/446
+      inreplace "script/build", "go1.5", "go#{Formula["go"].version}"
```

Comment:
Oh ugh, yeah, good catch. I guess `Utils.popen_read` on the `go` executable itself is an alternative.